crates/gitbutler-tauri/src/menu.rs

```diff
@@ -31,10 +31,30 @@ pub async fn menu_item_set_enabled(
 }
 
 pub fn build(package_info: &PackageInfo) -> Menu {
-    Menu::os_default(&package_info.name).add_submenu(Submenu::new(
-        "Project",
-        Menu::with_items([disabled_menu_item("project/settings", "Project Settings")]),
-    ))
+    extend_menu(Menu::os_default(&package_info.name))
+}
+
+fn extend_menu(menu: Menu) -> Menu {
+    let mut extended_menu = Menu::new();
+    for item in menu.items {
+        match item {
+            MenuEntry::Submenu(sm) => {
+                if sm.title == "View" {
+                    let mut custom_items = vec![
+                        disabled_menu_item("view/settings", "Project Settings"),
+                        CustomMenuItem::new("view/devtools", "Developer Tools").into(),
+                    ];
+                    custom_items.extend(sm.inner.items);
+                    extended_menu = extended_menu
+                        .add_submenu(Submenu::new("View", Menu::with_items(custom_items)));
+                } else {
+                    extended_menu = extended_menu.add_submenu(sm.clone());
+                }
+            }
+            _ => continue,
```

Comment:
```suggestion
        if let MenuEntry::Submentu(sm) = item {
                if sm.title == "View" {
                    let mut custom_items = vec![
                        disabled_menu_item("view/settings", "Project Settings"),
                        CustomMenuItem::new("view/devtools", "Developer Tools").into(),
                    ];
                    custom_items.extend(sm.inner.items);
                    extended_menu = extended_menu
                        .add_submenu(Submenu::new("View", Menu::with_items(custom_items)));
                } else {
                    extended_menu = extended_menu.add_submenu(sm.clone());
                }
            }
```

Probably not a 'valid' diff but you can match a single variant like this.